velox/type/Type.cpp

```diff
@@ -897,6 +899,14 @@ std::string IntervalDayTimeType::valueToString(int64_t value) const {
   return buf;
 }
 
+std::string IntervalYearMonthType::valueToString(int32_t value) const {
+  const int32_t years = value / kMonthInYear;
```

Comment:
What is the behavior if the input is a negative value ? Does -ve intervals have any special meaning like before or after ?

If -ve values is valid, then please add tests for those.

Reply:
What do you mean by "-ve intervals"?

Just updated the logic according to presto java worker to keep consistent